packages/cli/src/bin/ceramic.ts

```diff
@@ -47,6 +47,7 @@ program
     '--sync-override <string>',
     'Global forced mode for syncing all streams. One of: "prefer-cache", "sync-always", or "never-sync". Defaults to "prefer-cache". Deprecated.'
   )
+  .option('--composedb-enabled <boolean>', 'Run node with Compose DB indexing enabled.')
```

Comment:
we've had problems in the past with setting boolean flags to false.  I think the option parsing library we use will interpret the boolean as "true" if the flag is provided at all.  We might want to consider making this a `--disable-composedb` flag that doesn't take any arguments for simplicity

Reply:
OK, if there have been issues in the past with it I'll change the flag and address the downstream comments accordingly.

Reply:
talked about this with @oed and he's in alignment with this approach.  He suggests we should also change the config file option to disable-composedb so all 3 options are symmetrical